examples/xml/behat.yml

```diff
@@ -0,0 +1,18 @@
+default:
+  extensions:
+    Arachne\ServiceContainer\ArachneExtension:
+      base_url: http://urlecho.appspot.com
```

Comment:
Because http://echo.jsontest.com/ only echo json, I devided the example behat configuration into json and xml and used a xml echo service.